SU-HSDO/suhumsci
# fix(STN-994): scss for private page icon

# READY FOR REVIEW

## Summary
Corrects disappearing lock icon on hover within mega menu level 1 nav items.

## Need Review By (Date)
asap

## Urgency
high

## Steps to Test
1. Enable the mega menu in theme settings
2. Add a private page to the menu as a level 1 nav item
3. View the nav in both default masthead and dark masthead

- [x] Confirm the lock icon is visible and aligned consistently with how it is normally aligned on the standard menu
- [x] Confirm the lock icon is visible on hover
- [x] Confirm the lock icon is white when a dark masthead is enabled

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
